### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by dtrai2**

### CODE REVIEW

**Review by ppcad — APPROVED** [review]

**Comment by ppcad** [comment]
I realized that "Fix crashes of the generic adder if sql-based rules are processed without having set the sql config" can be removed form bugfixes again,  since the adder was only added with this release.  Could you please delete this line?

**Review by ppcad — CHANGES_REQUESTED** [review]
I realized that "Fix crashes of the generic adder if sql-based rules are processed without having set the sql config" can be removed form bugfixes again,  since the adder was only added with this release.  Could you please delete that line?

Edit: I see it was added as new feature in 2.0 and 3.0. The addition in 2.0 was a mistake.

**Comment by dtrai2** [comment]
So I should remove the line `* Add functionality to generic adder to add data from MySQL tables` in v2.0.0 Features and the line `* Fix crashes of the generic adder if sql-based rules are processed without having set the sql config` because there is no need for a fix if we just release this functionality in general?

**Comment by ppcad** [comment]
> So I should remove the line `* Add functionality to generic adder to add data from MySQL tables` in v2.0.0 Features and the line `* Fix crashes of the generic adder if sql-based rules are processed without having set the sql config` because there is no need for a fix if we just release this functionality in general?

Yes, please.

**Review by ppcad — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
...
+### Features
+### Improvements
+### Bugfixes
+### Breaking
+
+## v3.0.0
+
 ### Features
 
 * Add ability to add fields from SQL Database via in GenericAdder
```
[the remark this answers is not in the document]
Comment (dtrai2) [reply CHANGELOG.md:12]: Well the release has already happened. I sadly released first and then started to prepare the Changelog for the next release. So the changes we are doing now are actually preparations for the next release. Next time I should maybe make it the other way around. 
But it's okay, I will fix that mistake as well and then we have a clean version. I will also try to adjust the release notes from the actual github release then.

[the remark this answers is not in the document]
Comment (ppcad) [reply CHANGELOG.md]: Ah, I did not notice. Thanks for the changes.

**Pull request merged**